recipes/rabbitmq/f52bca4.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/sbin/rabbitmq-defaults b/sbin/rabbitmq-defaults
+index b845f74..95dccce 100755
+--- a/sbin/rabbitmq-defaults
++++ b/sbin/rabbitmq-defaults
+@@ -16,7 +16,7 @@
+ ##
+ 
+ ### next line potentially updated in package install steps
+-SYS_PREFIX=${RABBITMQ_HOME}
++SYS_PREFIX=${PREFIX}
```

Comment:
Okay. In that case you might want to change `${PREFIX}` to `${CONDA_PREFIX}` because `PREFIX` is only set in a conda build environment and not in a normal conda environment.